www/HW-03/css/styles.css

```diff
@@ -0,0 +1,132 @@
+body{
+    margin: 0;
+    padding: 0;
+}
+
+.redText {
+    color:red;
+}
+
+.greenText{
+  color:rgb(0,255,0);
+}
+
+.blueText{
+    color:#0000ff;
+    font-weight: bold;
+}
+
+.px1{
+    font-size:18px;
+}
+
+.px2
+{
+    font-size: 24px;
+    font-style: italic;
+}
+
+.brn{
+    background-color: brown;
+    color:white
+}
+
+#p1{
+line-height:32px;
+text-align:center
+}
+
+#p2{
+    text-transform: uppercase;
+    text-align: justify;
+    }
+
+.box01 {
+    width: 400px;
+    height: 300px;
+    border-color: purple;
+    border-style: solid;
+    border-width: 1px;
+    padding: 30px;
+}
+
+.box02 {
+    width: 500px;
+    height: 200px;
+    background-color: red;
+    margin:100px;
+
+}
+
+.non-vis1{
+   
+    display: none;
+}
+
+.box03{
+    width: 100%;
+    height: 150px;
+    background-color: blue;
+    padding:50px;
+    border-color:yellow;
+    border-style: solid;
+    border-width:10px;
+    box-sizing: border-box; 
+}
+
+.non-vis2{
+    visibility: hidden;
+}
+
+.box04{
+    width: 500px;
+    height: 200px;
+    background-color: green;
+    opacity: 0.5;
+
+}
+
+.box05{
+    width: 400px;
+    height: 300px;
+    background-color: lightblue;
+    position: relative;
+}
+
+.box06{
+    width: 50px;
+    height: 50px;
+    background-color: pink;
+    position: absolute;
+    top: 50px;
+    right: 10px; 
+}
+
+.but01 {
+    width: 300px;
+    height: 100px;
+}
+
+.but01:active{
+    color:blue;
+}
+
+.but02 {
+    width: 200px;
+    height: 50px;
+}
+
+.but02:hover{
+    background-color: brown;
+}
+
+.box07{
+    width: 200px;
+    height: 500px;
+    background-color: blue;
+    margin-top: 200px;
+}
+
+.box07:hover{
+    cursor:pointer;
```

Comment:
vlastnost cursor nemusi byt v :hover, protoze sama jiz implikuje to, ze element musi byt hovered aby to melo efekt